src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -645,6 +664,26 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 		}
 	}
 
+	// A funtion that filters out specifically the `value` property of the attachment.
+	private getFilteredEntry(query: string, inputState: IChatInputState): IChatHistoryEntry {
+		const attachmentsWithoutImageValues = inputState.chatContextAttachments?.map(attachment => {
+			if (attachment.isImage && attachment.references?.length && attachment.value) {
```

Comment:
Is there some case where an image attachment doesn't have references, and then the image data will still be saved in history?

Reply:
there shouldn't be case where images don't have references (if it comes from web and is an image, it is okay because the `value` will be a URL)